testing/astromenace/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=astromenace
+pkgver=1.4.2
+pkgrel=0
+pkgdesc="Hardcore 3D space shooter with spaceship upgrade possibilities"
+url="http://www.viewizard.com/astromenace/index_linux.php"
+arch="x86_64"
+license="GPL3"
+depends="sdl2 libjpeg-turbo libvorbis glu freetype libxinerama sdl2 openal-soft"
+makedepends="cmake mesa"
+source="$pkgname-$pkgver.zip::https://github.com/viewizard/astromenace/archive/master.zip"
```

Comment:
not valid, this is not a versioned release, please use a versioned release or ask upstream for one